### PR TITLE
DXCDT-276: Update commands help text

### PR DESCRIPTION
**Pull request by sergiught**
### 🔧 Changes

In this PR we are updating the help text to offer additional information and to make the information display in a consistent manner across all commands.

To note that we are not re-generating the documentation to reduce the amount of changes displayed in this PR. 

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/display/apis.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (v *apiTableView) Object() interface{} {
 }
 
 func (r *Renderer) ApiList(apis []*management.ResourceServer) {
-	resource := "APIs"
```
Comment (sergiught) [inline internal/display/apis.go:71]: We are also making all the headings that appear after the tenant's name lowercase for consistency.

File: internal/cli/templates.go [hunk 2]
```diff
@@ -153,31 +153,21 @@ func resourceUsageTemplate() string {
 
 Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
 `,
-		ansi.Faint("Usage:"),
```
Comment (sergiught) [inline internal/cli/templates.go:156]: We're making these headings bold so they are more easily displayed for the user.

File: internal/cli/actions.go [hunk 3]
```diff
@@ -171,12 +174,14 @@ func createActionCmd(cli *cli) *cobra.Command {
...
 		Short: "Create a new action",
-		Long:  "Create a new action.",
-		Example: `auth0 actions create 
-auth0 actions create --name myaction
-auth0 actions create -n myaction --trigger post-login
-auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
-auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -s "API_KEY=value" -s "SECRET=value`,
+		Long: "Create a new action.\n\n" +
+			"To create an action interactively, use `auth0 actions create` with no arguments.\n\n" +
+			"To create an action non-interactively, supply the action name, trigger, secrets and dependencies.",
```
Comment (Widcket) [inline internal/cli/actions.go:179]:
```suggestion
			"To create an action non-interactively, supply the action name, trigger, secrets and dependencies through the flags.",
```

File: internal/cli/actions.go [hunk 4]
```diff
@@ -171,12 +174,14 @@ func createActionCmd(cli *cli) *cobra.Command {
...
 		Args:  cobra.NoArgs,
 		Short: "Create a new action",
-		Long:  "Create a new action.",
-		Example: `auth0 actions create 
-auth0 actions create --name myaction
-auth0 actions create -n myaction --trigger post-login
-auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
-auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -s "API_KEY=value" -s "SECRET=value`,
+		Long: "Create a new action.\n\n" +
+			"To create an action interactively, use `auth0 actions create` with no arguments.\n\n" +
```
Comment (Widcket) [inline internal/cli/actions.go:178]:
```suggestion
			"To create an action interactively, use `auth0 actions create` with no flags.\n\n" +
```

File: internal/cli/actions.go [hunk 5]
```diff
@@ -382,9 +390,12 @@ func deleteActionCmd(cli *cli) *cobra.Command {
 		Use:   "delete",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Delete an action",
-		Long:  "Delete an action.",
-		Example: `auth0 actions delete 
-auth0 actions delete <id>`,
+		Long: "Delete an action.\n\n" +
+			"To delete an action interactively, use `auth0 actions delete` with no arguments.\n\n" +
```
Comment (Widcket) [inline internal/cli/actions.go:394]:
```suggestion
			"To delete an action interactively, use `auth0 actions delete` with no flags.\n\n" +
```

File: internal/cli/actions.go [hunk 6]
```diff
@@ -253,12 +258,15 @@ func updateActionCmd(cli *cli) *cobra.Command {
...
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Update an action",
-		Long:  "Update an action.",
-		Example: `auth0 actions update <id> 
-auth0 actions update <id> --name myaction
-auth0 actions update <id> -n myaction --trigger post-login
-auth0 actions update <id> -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
-auth0 actions update <id> -n myaction -t post-login -d "lodash=4.0.0" -s "API_KEY=value" -s "SECRET=value`,
+		Long: "Update an action.\n\n" +
+			"To update an action interactively, use `auth0 actions update` with no arguments.\n\n" +
```
Comment (Widcket) [inline internal/cli/actions.go:262]:
```suggestion
			"To update an action interactively, use `auth0 actions update` with no flags.\n\n" +
```

File: internal/cli/actions.go [hunk 7]
```diff
@@ -171,12 +174,14 @@ func createActionCmd(cli *cli) *cobra.Command {
...
-auth0 actions create -n myaction --trigger post-login
-auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
-auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -s "API_KEY=value" -s "SECRET=value`,
+		Long: "Create a new action.\n\n" +
+			"To create an action interactively, use `auth0 actions create` with no arguments.\n\n" +
+			"To create an action non-interactively, supply the action name, trigger, secrets and dependencies.",
+		Example: `  auth0 actions create
+  auth0 actions create --name myaction
+  auth0 actions create -n myaction --trigger post-login
+  auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
```
Comment (Widcket) [inline internal/cli/actions.go:183]:
```suggestion
  auth0 actions create -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=9.0.0"
```

File: internal/cli/actions.go [hunk 8]
```diff
@@ -253,12 +258,15 @@ func updateActionCmd(cli *cli) *cobra.Command {
...
-auth0 actions update <id> -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
-auth0 actions update <id> -n myaction -t post-login -d "lodash=4.0.0" -s "API_KEY=value" -s "SECRET=value`,
+		Long: "Update an action.\n\n" +
+			"To update an action interactively, use `auth0 actions update` with no arguments.\n\n" +
+			"To update an action non-interactively, supply the action id, name, trigger, secrets and " +
+			"dependencies through the flags.",
+		Example: `  auth0 actions update <id> 
+  auth0 actions update <id> --name myaction
+  auth0 actions update <id> -n myaction --trigger post-login
+  auth0 actions update <id> -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=8.0.0"
```
Comment (Widcket) [inline internal/cli/actions.go:268]:
```suggestion
  auth0 actions update <id> -n myaction -t post-login -d "lodash=4.0.0" -d "uuid=9.0.0"
```

File: internal/cli/actions.go (hunk 4 above)
Comment (willvedd) [inline internal/cli/actions.go:178]:
```suggestion
			"To create interactively, use `auth0 actions create` with no arguments.\n\n" +
```

File: internal/cli/actions.go (hunk 3 above)
Comment (willvedd) [inline internal/cli/actions.go:179]:
```suggestion
			"To create non-interactively, supply the action name, trigger, secrets and dependencies.",
```

File: internal/cli/apis.go [hunk 9]
```diff
@@ -58,9 +58,11 @@ var (
 
 func apisCmd(cli *cli) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "apis",
-		Short:   "Manage resources for APIs",
-		Long:    "Manage resources for APIs.",
+		Use:   "apis",
+		Short: "Manage resources for APIs",
+		Long: "An API is an entity that represents an external resource, capable of " +
```
Comment (willvedd) [inline internal/cli/apis.go:63]:
```suggestion
		Long: "Manage resources for APIs. An API is an entity that represents an external resource, capable of " +
```

File: internal/cli/apis.go [hunk 10]
```diff
@@ -198,12 +201,15 @@ func createApiCmd(cli *cli) *cobra.Command {
...
 		Args:  cobra.NoArgs,
 		Short: "Create a new API",
-		Long:  "Create a new API.",
-		Example: `auth0 apis create 
-auth0 apis create --name myapi
-auth0 apis create -n myapi --identifier http://my-api
-auth0 apis create -n myapi --token-expiration 6100
-auth0 apis create -n myapi -e 6100 --offline-access=true`,
+		Long: "Create a new API.\n\n" +
+			"To create an API interactively, use `auth0 apis create` with no arguments.\n\n" +
```
Comment (Widcket) [inline internal/cli/apis.go:205]:
```suggestion
			"To create an API interactively, use `auth0 apis create` with no flags.\n\n" +
```

File: internal/cli/apis.go [hunk 11]
```diff
@@ -279,12 +285,15 @@ func updateApiCmd(cli *cli) *cobra.Command {
...
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Update an API",
-		Long:  "Update an API.",
-		Example: `auth0 apis update 
-auth0 apis update <id|audience> 
-auth0 apis update <id|audience> --name myapi
-auth0 apis update -n myapi --token-expiration 6100
-auth0 apis update -n myapi -e 6100 --offline-access=true`,
+		Long: "Update an API.\n\n" +
+			"To update an API interactively, use `auth0 apis update` with no arguments.\n\n" +
```
Comment (Widcket) [inline internal/cli/apis.go:289]:
```suggestion
			"To update an API interactively, use `auth0 apis update` with no flags.\n\n" +
```

File: internal/cli/apis.go [hunk 12]
```diff
@@ -279,12 +285,15 @@ func updateApiCmd(cli *cli) *cobra.Command {
...
+			"To update an API non-interactively, supply the name, identifier, scopes, " +
+			"token lifetime and whether to allow offline access.",
```
Comment (Widcket) [inline internal/cli/apis.go:291]:
```suggestion
			"token lifetime and whether to allow offline access through the flags.",
```

File: internal/cli/apis.go [hunk 13]
```diff
@@ -378,9 +387,12 @@ func deleteApiCmd(cli *cli) *cobra.Command {
 		Use:   "delete",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Delete an API",
-		Long:  "Delete an API.",
-		Example: `auth0 apis delete 
-auth0 apis delete <id|audience>`,
+		Long: "Delete an API.\n\n" +
+			"To delete an API interactively, use `auth0 apis delete` with no arguments.\n\n" +
```
Comment (Widcket) [inline internal/cli/apis.go:391]:
```suggestion
			"To delete an API interactively, use `auth0 apis delete` with no flags.\n\n" +
```

File: internal/cli/apis.go [hunk 14]
```diff
@@ -473,9 +485,10 @@ func listScopesCmd(cli *cli) *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List the scopes of an API",
-		Long:    "List the scopes of an API.",
-		Example: `auth0 apis scopes list 
-auth0 apis scopes ls <id|audience>`,
+		Long:    "List the scopes of an API. To add scopes, try running: `auth0 apis update <id|audience> -s <scopes>`.",
```
Comment (willvedd) [inline internal/cli/apis.go:488]:
```suggestion
		Long:    "List the scopes of an API. To update scopes, try running: `auth0 apis update <id|audience> -s <scopes>`.",
```

File: internal/cli/apps.go [hunk 15]
```diff
@@ -154,11 +156,13 @@ func useAppCmd(cli *cli) *cobra.Command {
...
 	cmd := &cobra.Command{
-		Use:     "use",
-		Args:    cobra.MaximumNArgs(1),
-		Short:   "Choose a default application for the Auth0 CLI",
-		Long:    "Specify your preferred application for interaction with the Auth0 CLI.",
-		Example: "auth0 apps use <client-id>",
+		Use:   "use",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Choose a default application for the Auth0 CLI",
+		Long:  "Specify your preferred application for interaction with the Auth0 CLI.",
```
Comment (willvedd) [inline internal/cli/apps.go:162]:
```suggestion
		Long:  "Specify the default application used when running other commands. Specifically when downloading quickstarts and testing Universal login flow.",
```

File: internal/cli/apps.go [hunk 16]
```diff
@@ -254,9 +258,10 @@ func showAppCmd(cli *cli) *cobra.Command {
 		Use:   "show",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Show an application",
-		Long:  "Show an application.",
-		Example: `auth0 apps show 
-auth0 apps show <id>`,
+		Long:  "Display the name, description, type, and other information about an application.",
```
Comment (willvedd) [inline internal/cli/apps.go:261]:
```suggestion
		Long:  "Display the name, description, app type, and other information about an application.",
```

File: internal/cli/log_streams.go [hunk 17]
```diff
@@ -752,11 +765,12 @@ func openLogStreamsCmd(cli *cli) *cobra.Command {
...
 
 	cmd := &cobra.Command{
-		Use:     "open",
-		Args:    cobra.MaximumNArgs(1),
-		Short:   "Open log stream settings page in the Auth0 Dashboard",
-		Long:    "Open log stream settings page in the Auth0 Dashboard.",
-		Example: "auth0 logs streams open <id>",
+		Use:   "open",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Open the details page of a log stream",
```
Comment (willvedd) [inline internal/cli/log_streams.go:770]:
```suggestion
		Short: "Open the settings page of a log stream",
```

File: internal/cli/actions.go [hunk 18]
```diff
@@ -471,11 +487,12 @@ func openActionCmd(cli *cli) *cobra.Command {
...
 	cmd := &cobra.Command{
-		Use:     "open",
-		Args:    cobra.MaximumNArgs(1),
-		Short:   "Open action details page in the Auth0 Dashboard",
-		Long:    "Open action details page in the Auth0 Dashboard.",
-		Example: "auth0 actions open <id>",
+		Use:   "open",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Open the details page of an action",
+		Long:  "Open an action's details page in the Auth0 Dashboard.",
```
Comment (willvedd) [inline internal/cli/actions.go:493]: **Reminder:** Update usage of "details page" to "settings page" for consistency.

File: internal/cli/log_streams.go [hunk 19]
```diff
@@ -263,12 +266,15 @@ func createLogStreamCmd(cli *cli) *cobra.Command {
...
-auth0 logs streams create -n test-splunk -t splunk --splunk-domain demo.splunk.com --splunk-token 12a34ab5-c6d7-8901-23ef-456b7c89d0c1 --splunk-port 8080 --splunk-secure=true`,
+		Long: "Create a new log stream.\n\n" +
+			"To create a log stream interactively, use `auth0 logs streams create` with no arguments.\n\n" +
+			"To create a log stream non-interactively, supply the log stream name, type and other information " +
+			"through the flags.",
+		Example: `  auth0 logs streams create
+  auth0 logs streams create -n mylogstream -t http --http-type application/json --http-format JSONLINES --http-auth 1343434
+  auth0 logs streams create -n mydatadog -t datadog --datadog-key 9999999 --datadog-id us
+  auth0 logs streams create -n myeventbridge -t eventbridge --eventbridge-id 999999999999 --eventbridge-region us-east-1
+  auth0 logs streams create -n test-splunk -t splunk --splunk-domain demo.splunk.com --splunk-token 12a34ab5-c6d7-8901-23ef-456b7c89d0c1 --splunk-port 8080 --splunk-secure=true`,
```
Comment (willvedd) [inline internal/cli/log_streams.go:277]:
```suggestion
		Example: `  auth0 logs streams create
  # Custom Webhook
  auth0 logs streams create -n mylogstream -t http --http-type application/json --http-format JSONLINES --http-auth 1343434
  # Datadog 
  auth0 logs streams create -n mydatadog -t datadog --datadog-key 9999999 --datadog-id us
  # EventBridge
  auth0 logs streams create -n myeventbridge -t eventbridge --eventbridge-id 999999999999 --eventbridge-region us-east-1
  # Splunk
  auth0 logs streams create -n test-splunk -t splunk --splunk-domain demo.splunk.com --splunk-token 12a34ab5-c6d7-8901-23ef-456b7c89d0c1 --splunk-port 8080 --splunk-secure=true`,
```

File: internal/cli/organizations.go [hunk 20]
```diff
@@ -110,11 +111,11 @@ func listOrganizationsCmd(cli *cli) *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    cobra.NoArgs,
 		Short:   "List your organizations",
-		Long: `List your existing organizations. To create one try:
-auth0 orgs create`,
-		Example: `auth0 orgs list
-auth0 orgs ls
-auth0 orgs ls -n 100`,
+		Long:    "List your existing organizations. To create one, try running: `auth0 orgs create`.",
```
Comment (willvedd) [inline internal/cli/organizations.go:114]:
```suggestion
		Long:    "List your existing organizations. To create one, run: `auth0 orgs create`.",
```

File: internal/cli/organizations.go [hunk 21]
```diff
@@ -497,9 +508,10 @@ func listMembersOrganizationCmd(cli *cli) *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List members of an organization",
-		Long:    "List members of an organization.",
-		Example: `auth0 orgs members list
-auth0 orgs members ls <id>`,
+		Long:    "List the existing members of an organization.",
```
Comment (willvedd) [inline internal/cli/organizations.go:511]:
```suggestion
		Long:    "List the members of an organization.",
```

File: internal/cli/quickstarts.go [hunk 22]
```diff
@@ -96,8 +97,8 @@ func downloadQuickstartCmd(cli *cli) *cobra.Command {
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Download a Quickstart sample app for a specific tech stack",
 		Long:  "Download a Quickstart sample app for a specific tech stack.",
```
Comment (willvedd) [inline internal/cli/quickstarts.go:99]:
```suggestion
		Long:  "Download a Quickstart sample application for that’s already configured for your Auth0 application. Many different tech stacks available.",
```

File: internal/cli/rules.go [hunk 23]
```diff
@@ -454,7 +463,7 @@ func disableRuleCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "Disable a rule",
 		Long:    "Disable a rule.",
-		Example: `auth0 rules disable <id>`,
+		Example: `  auth0 rules disable <id>`,
```
Comment (willvedd) [inline internal/cli/rules.go:466]:
```suggestion
		Example: `  auth0 rules disable
  auth0 rules disable <id>`,
```

File: internal/cli/rules.go [hunk 24]
```diff
@@ -400,7 +409,7 @@ func enableRuleCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "Enable a rule",
 		Long:    "Enable a rule.",
-		Example: `auth0 rules enable <id>`,
+		Example: `  auth0 rules enable <id>`,
```
Comment (willvedd) [inline internal/cli/rules.go:412]:
```suggestion
		Example: `  auth0 rules enable
  auth0 rules enable <id>`,
```

File: internal/cli/universal_login.go [hunk 25]
```diff
@@ -260,11 +266,12 @@ auth0 universal-login update -a "#FF4F40" -b "#2A2E35" --logo "https://example.c
...
 	cmd := &cobra.Command{
-		Use:     "show",
-		Args:    cobra.NoArgs,
-		Short:   "Display the custom template for Universal Login",
-		Long:    "Display the custom template for Universal Login.",
-		Example: "auth0 universal-login templates show",
+		Use:   "show",
+		Args:  cobra.NoArgs,
+		Short: "Display the custom template for Universal Login",
+		Long:  "Display the custom template for Universal Login.",
```
Comment (willvedd) [inline internal/cli/universal_login.go:272]:
```suggestion
		Long:  "Display the custom template for the Universal Login experience.",
```

**Review by willvedd — COMMENTED** [review]
Reviewed w/ Sergiu, many minor comments but otherwise good.

**Review by willvedd — APPROVED** [review]
We combed through this together. LGTM 👍

File: internal/cli/organizations.go [hunk 26]
```diff
@@ -408,9 +415,12 @@ func deleteOrganizationCmd(cli *cli) *cobra.Command {
...
 		Short: "Delete an organization",
-		Long:  "Delete an organization.",
-		Example: `auth0 orgs delete
-auth0 orgs delete <id>`,
+		Long: "Delete an organization.\n\n" +
+			"To delete interactively, use `auth0 orgs delete` with no arguments.\n\n" +
+			"To delete non-interactively, supply the organization id and the `--force` " +
+			"flag to skip confirmation.",
+		Example: `  auth0 orgs delete
+  auth0 orgs delete <id>`,
```
Comment (Widcket) [inline internal/cli/organizations.go:423]: Consider adding the `auth0 orgs delete <id> --force` example in a future PR, like it is in other commands.

File: internal/cli/apps.go [hunk 27]
```diff
@@ -298,9 +304,12 @@ func deleteAppCmd(cli *cli) *cobra.Command {
...
 		Short: "Delete an application",
-		Long:  "Delete an application.",
-		Example: `auth0 apps delete 
-auth0 apps delete <id>`,
+		Long: "Delete an application.\n\n" +
+			"To delete interactively, use `auth0 apps delete` with no arguments.\n\n" +
+			"To delete non-interactively, supply the application id and the `--force` " +
+			"flag to skip confirmation.",
+		Example: `  auth0 apps delete 
+  auth0 apps delete <id>`,
```
Comment (Widcket) [inline internal/cli/apps.go:312]: Consider adding the `auth0 apps delete <id> --force` example in a future PR, like it is in other commands.

File: internal/cli/users.go [hunk 28]
```diff
@@ -322,7 +324,9 @@ func deleteUserCmd(cli *cli) *cobra.Command {
 		Use:   "delete",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Delete a user",
-		Long:  "Delete a user.",
+		Long: "Delete a user.\n\n" +
+			"To delete interactively, use `auth0 users delete` with no arguments.\n\n" +
+			"To delete non-interactively, supply the user id and the `--force` flag to skip confirmation.",
 		Example: `auth0 users delete 
 auth0 users delete <id>`,
```
Comment (Widcket) [inline internal/cli/users.go:331]: Consider adding the `auth0 users delete <id> --force` example in a future PR, like it is in other commands.

File: internal/cli/organizations.go [hunk 29]
```diff
@@ -611,8 +626,8 @@ func listMembersRolesOrganizationCmd(cli *cli) *cobra.Command {
 		Args:  cobra.MaximumNArgs(1),
 		Short: "List organization members for a role",
 		Long:  "List organization members that have a given role assigned to them.",
-		Example: `auth0 orgs roles members list
-auth0 orgs roles members list <org id> --role-id role`,
+		Example: `  auth0 orgs roles members list
+  auth0 orgs roles members list <org id> --role-id role`,
```
Comment (Widcket) [inline internal/cli/organizations.go:630]: Consider adding in a future PR an example with the `ls` alias, like in other `list` commands.

File: internal/cli/users.go [hunk 30]
```diff
@@ -516,7 +522,7 @@ func listUserBlocksCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List brute-force protection blocks for a given user",
 		Long:    "List brute-force protection blocks for a given user.",
-		Example: "auth0 users blocks list <user-id>",
+		Example: `  auth0 users blocks list <user-id>`,
```
Comment (Widcket) [inline internal/cli/users.go:525]: Consider adding in a future PR an example with the `ls` alias, like in other `list` commands.

File: internal/cli/apis.go [hunk 31]
```diff
@@ -99,11 +101,11 @@ func listApisCmd(cli *cli) *cobra.Command {
...
 		Short:   "List your APIs",
-		Long: `List your existing APIs. To create one try:
-auth0 apis create`,
-		Example: `auth0 apis list
-auth0 apis ls
-auth0 apis ls -n 100`,
+		Long:    "List your existing APIs. To create one, run: `auth0 apis create`.",
+		Example: `  auth0 apis list
+  auth0 apis ls
+  auth0 actions ls --json
```
Comment (Widcket) [inline internal/cli/apis.go:107]: This should be `auth0 apis ls --json` (`apis` instead of `actions`).

File: internal/cli/organizations.go [hunk 32]
```diff
@@ -497,9 +508,10 @@ func listMembersOrganizationCmd(cli *cli) *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List members of an organization",
-		Long:    "List members of an organization.",
-		Example: `auth0 orgs members list
-auth0 orgs members ls <id>`,
+		Long:    "List the members of an organization.",
+		Example: `  auth0 orgs members list
+  auth0 orgs members ls <id>
+  auth0 orgs members ls <id> --json`,
```
Comment (Widcket) [inline internal/cli/organizations.go:514]: As an org is likely to have many members, consider in a future PR adding an example with the `-n` flag, like in other similar commands.

File: internal/cli/organizations.go (hunk 29 above)
Comment (Widcket) [reply internal/cli/organizations.go:630]: Also, as an org is likely to have many members that have a particular role, consider in a future PR adding an example with the `-n` flag, like in other similar commands.

File: internal/cli/organizations.go [hunk 33]
```diff
@@ -553,8 +566,9 @@ func listRolesOrganizationCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List roles of an organization",
 		Long:    "List roles assigned to members of an organization.",
-		Example: `auth0 orgs roles list
-auth0 orgs roles ls <id>`,
+		Example: `  auth0 orgs roles list
+  auth0 orgs roles ls <id>
+  auth0 orgs roles ls <id> --json`,
```
Comment (Widcket) [inline internal/cli/organizations.go:571]: As an org is likely to have many roles assigned to its members, consider in a future PR adding an example with the `-n` flag, like in other similar commands.

File: internal/cli/roles_permissions.go [hunk 34]
```diff
@@ -56,10 +56,10 @@ func listRolePermissionsCmd(cli *cli) *cobra.Command {
...
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List permissions defined within a role",
-		Long: `List existing permissions defined in a role. To add a permission try:
-auth0 roles permissions add <role-id>`,
-		Example: `auth0 roles permissions list <role-id>
-auth0 roles permissions ls`,
+		Long:    "List existing permissions defined in a role. To add a permission, run: `auth0 roles permissions add`.",
+		Example: `  auth0 roles permissions list
+  auth0 roles permissions list <role-id>
+  auth0 roles permissions ls --json`,
```
Comment (Widcket) [inline internal/cli/roles_permissions.go:62]: As a given role is likely to have many permissions, consider in a future PR adding an example with the `-n` flag, like in other similar commands.

File: internal/cli/apps.go [hunk 35]
```diff
@@ -254,9 +259,10 @@ func showAppCmd(cli *cli) *cobra.Command {
 		Use:   "show",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Show an application",
-		Long:  "Show an application.",
-		Example: `auth0 apps show 
-auth0 apps show <id>`,
+		Long:  "Display the name, description, app type, and other information about an application.",
+		Example: `  auth0 apps show
+  auth0 apps show --json
+  auth0 apps show <id>`,
```
Comment (Widcket) [inline internal/cli/apps.go:265]: This one should probably go first.

File: internal/cli/attack_protection_breached_password_detection.go [hunk 36]
```diff
@@ -76,12 +79,13 @@ func breachedPasswordDetectionCmd(cli *cli) *cobra.Command {
...
-		Short:   "Show breached password detection settings",
-		Long:    "Show breached password detection settings.",
-		Example: `auth0 protection breached-password-detection show`,
-		RunE:    showBreachedPasswordDetectionCmdRun(cli),
+		Use:   "show",
+		Args:  cobra.NoArgs,
+		Short: "Show breached password detection settings",
+		Long:  "Display the current breached password detection settings.",
+		Example: `  auth0 protection breached-password-detection show
+  auth0 ap bpd show --json`,
```
Comment (Widcket) [inline internal/cli/attack_protection_breached_password_detection.go:87]: Consider in a future PR adding an (additional) example without the `--json` flag (table view), like in other `show` commands.

File: internal/cli/users.go (hunk 30 above)
Comment (Widcket) [reply internal/cli/users.go:525]: Consider as well adding an example with the `--json` flag, like in other similar commands.

File: internal/cli/organizations.go (hunk 29 above)
Comment (Widcket) [reply internal/cli/organizations.go:630]: Consider as well adding an example with the `--json` flag, like other similar commands.

File: internal/cli/apis.go [hunk 37]
```diff
@@ -152,9 +154,10 @@ func showApiCmd(cli *cli) *cobra.Command {
 		Use:   "show",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "Show an API",
-		Long:  "Show an API.",
-		Example: `auth0 apis show 
-auth0 apis show <id|audience>`,
+		Long:  "Display the name, scopes, token lifetime, and other information about an API.",
```
Comment (Widcket) [inline internal/cli/apis.go:157]: Some long descriptions start with the short one, others don't. Consider sticking to a single style in a future PR.

**Pull request merged**